dev/release/generate-changelog.py

```diff
@@ -44,6 +44,7 @@ def generate_changelog(repo, repo_name, tag1, tag2, version):
     unique_pulls = []
     all_pulls = []
     for commit in comparison.commits:
+        print(".", file=sys.stderr, end=" ", flush=True) # progress indicator
```

Comment:
👍

Reply:
Yeah, sorry I should have commented on that one -- it is due to the fact I am on a relatively slow internet connection and it wasn't clear to me the script was actually doing anything without this feedback.